### PR TITLE
the one item designed to hard counter kudzu no longer takes 1/2 of your stamina bar to take down one block of fireproofing kudzu

### PR DESCRIPTION
**Pull request by silicons**
1.25 per tile hit.

### CODE REVIEW

**Comment by lolman360** [comment]
black gar glasses from a hacked autodrobe are better than the scythe and it's sad

File: code/modules/hydroponics/hydroitemdefines.dm [hunk 1]
```diff
@@ -115,6 +115,7 @@
 	slot_flags = ITEM_SLOT_BACK
 	attack_verb = list("chopped", "sliced", "cut", "reaped")
 	hitsound = 'sound/weapons/bladeslice.ogg'
+	total_mass = 1.25
```
Comment (deathride58) [inline code/modules/hydroponics/hydroitemdefines.dm]: as much stamina cost as a screwdriver for 2.6x the amount of damage unsharpened screwdrivers do against normal body parts? you could probably buff scythes by making kudzu cutting restore stamina for each kudzu tile cut instead

Comment (silicons) [reply code/modules/hydroponics/hydroitemdefines.dm]: lol im dumb :+1: kk

**Review by deathride58 — APPROVED** [review]
👍

**Pull request merged**